src/api/users.ts

```diff
@@ -34,3 +34,13 @@ export const updateUser = async (id: number, data: FormData): Promise<UserReadSe
     throw error;
   }
 };
+export const filterUserList = async (username: string): Promise<UserReadSerializer> => {
```

Comment:
```suggestion
export const searchUser = async (q: string): Promise<UserReadSerializer> => {
```

As mentioned in the BE PR [feedback](https://github.com/thenewboston-developers/thenewboston-Backend/pull/210#pullrequestreview-2105584770), this `q` can be anything from the `search_fields`.

This way, we can have a generic searching